website/docs/apps/index.md

```diff
@@ -17,10 +17,6 @@ Here is a list of apps that are generally good to use and can come in handy in d
 
 ![filemerge](/filemerge.png)
 
-[Kaleidoscope](https://apps.apple.com/us/app/kaleidoscope/id587512244): Another tool for reviewing and merging text, images and folders. Its beautiful user interface and great image diffing capabilities are what sets it apart from other tools. Available for a trial period of 15 days and then it requires payment.
-
-![Kaleidoscope](/kaleidoscope.png)
```

Comment:
I also fixed all the broken links by removing them, I accidentally put them in the same commit as fixing the linting issues 🤦 But yeah we should no longer have any broken links!